RUL-448: add text and new_line options in concatenate action rule (#1504)
akeneo/pim-docs: manipulate_pim_data/rule/general_information_on_rule_format.rst

Concatenate
___________

> This action concatenates at least two sources into a single value. A source can be an attribute value, a text or a line break.

Reviewer: not sure about "text", to check with Camille

Concatenate
___________

This action concatenates at least two blocks into a single value. A block can be an attribute value or a text. You can also use the specific *new_line* block to start a new line.

> The possible attribute types are:

Reviewer 2: Why did you remove the "source" word?

Author: Camille B wanted to

> The possible block attribute types are:

Reviewer 3: ```suggestion
The possible attribute types are:
```

 - text
 - textarea

> By default a space is added between two source attribute values (not between a source attribute and a source text, or between two source texts). You can avoid this behavior by adding an empty text between the two source attribute values.

Reviewer: ```suggestion
By default, a space is added between two attribute value blocks (not between an attribute and a text block, or between two text blocks). You can avoid this behavior by adding an empty text between the two attribute value blocks.
```

| from | List of sets for all sources.                                                                                                                                              |
|      |                                                                                                                                                                            |
|      | A source must contain exactly one single key among the following ones, plus some extra keys defined further:                                                               |
|      |                                                                                                                                                                            |
|      | - field: attribute code.                                                                                                                                                   |
|      | - text: a specific text.                                                                                                                                                   |

> |      | - new_line: no specific value (e.g. ``new_line: ~``). Only available if the target attribute is a text area.                                                               |

Reviewer 4: `new_line: ~` is not an example; it's the only way to do, no ?

Author: you can use `new_line: true` or `new_line: "I love raclette"` it will work

.. tip::

>     For instance, to concatenate the "Brand: " text, the brand (non-localizable and non-scopable), a line break, the "Model: " text and the model in en_US locale into the description value in en_US locale, the action will be as follows:

Reviewer 2: ```suggestion
    For instance, to concatenate the "Brand: " text, the brand attribute (non-localizable and non-scopable), a line break, the "Model: " text and the model attribute in en_US locale into the description value in en_US locale, the action will be as follows:
```
